### PR TITLE
Update temporary_academic_staff.json

### PR DESCRIPTION
**Pull request by chathurawimalasiri**

### CODE REVIEW

**Comment by ishanfdo18098** [comment]
Please fix the email

**Comment by chathurawimalasiri** [comment]
@ishanfdo18098 Thanks ishan. Done

**Comment by ishanfdo18098** [comment]
And the merge conflict

**Comment by chathurawimalasiri** [comment]
Done

File: _data/temporary_academic_staff.json [hunk 1]
```diff
@@ -299,5 +299,16 @@
...
+    "staff_name": "Chathura Wimalasiri",
+    "text_below_name": "Temporary Instructor",
+    "url_image": "images/students/e18/E18402.jpg",
+    "email": "e18402@eng.pdn.ac.lk",
+    "url_profile": "/students/e18/402",
+    "linkedin": "https://www.linkedin.com/in/chathura-wimalasiri-b0637b1a1",
+    "github": "https://github.com/chathurawimalasiri",
+    "joined_date": "2024-10-01",
+    "left_date": ""
+  },
```
Comment (ishanfdo18098) [inline _data/temporary_academic_staff.json:313]: Please remove the comma here

**Pull request merged**